### PR TITLE
🎃 Rename LoanToken2 to FixedTermLoan

### PR DESCRIPTION
**Pull request by vanruch**

### CODE REVIEW

File: contracts/truefi2/FixedTermLoanAgency.sol [hunk 1]
```diff
@@ -353,7 +353,7 @@ contract FixedTermLoanAgency is IFixedTermLoanAgency, UpgradeableClaimable {
         uint256 apy = rate(pool, borrower, amount, term);
         require(apy <= _maxApy, "FixedTermLoanAgency: Calculated apy is higher than max apy");
 
-        ILoanToken2 loanToken = loanFactory.createLoanToken(pool, borrower, amount, term, apy);
+        IFixedTermLoan loanToken = loanFactory.createLoanToken(pool, borrower, amount, term, apy);
```
Comment (skalermo) [inline contracts/truefi2/FixedTermLoanAgency.sol:356]: Are we going to change function names like here, and also event and variable names which have "loanToken" in them?

Comment (vanruch) [reply contracts/truefi2/FixedTermLoanAgency.sol:356]: I'm against changing event, we can consider changing function name. I'd stick with current one as it pairs with createDebtToken well, but I see arguments for change too

Comment (MiksuJak) [reply contracts/truefi2/FixedTermLoanAgency.sol:356]: Why not rename the events? This contract hasn't even gone live yet

**Review by MiksuJak — APPROVED** [review]

File: contracts/truefi2/FixedTermLoanAgency.sol (hunk 1 above)
Comment (vanruch) [reply contracts/truefi2/FixedTermLoanAgency.sol:356]: I think Roman meant LoanFactory event

**Review by skalermo — APPROVED** [review]

**Pull request merged**